tests/e2e/suite_test.go

```diff
@@ -61,13 +55,12 @@ func TestE2E(t *testing.T) {
 	// Run tests through the Ginkgo runner with output to console + JUnit for Jenkins
 	var r []Reporter
 	if framework.TestContext.ReportDir != "" {
+		// Create the directory if it doesn't already exists
+		// NOTE: junit report can be simply created by executing your tests with the new --junit-report flags instead.
```

Comment:
"can be simply created by executing your tests with the new --junit-report flags instead" -> "can be created with new --junit-report flag"

Reply:
Done.